Running Java benchmark MGopt, class D, on 1 servers (np=4), sm,self

SLURM job ID: 775668

Servers:
mpi021
mpi021
mpi021
mpi021

mpirun --mca btl sm,self --bind-to core --map-by node -np 4 java NPB_MPJ.MGopt class=D

Start: Sat May 31 07:40:15 PDT 2014
===============================================================================
Exception in thread "main" java.lang.NegativeArraySizeException
Exception in thread "main" java.lang.NegativeArraySizeException
Exception in thread "main" java.lang.NegativeArraySizeException
Exception in thread "main" java.lang.NegativeArraySizeException
	at NPB_MPJ.MGopt.runBenchmark(MGopt.java:198)
	at NPB_MPJ.MGopt.main(MGopt.java:113)	at NPB_MPJ.MGopt.runBenchmark(MGopt.java:198)
	at NPB_MPJ.MGopt.main(MGopt.java:113)
	at NPB_MPJ.MGopt.runBenchmark(MGopt.java:198)
	at NPB_MPJ.MGopt.main(MGopt.java:113)
	at NPB_MPJ.MGopt.runBenchmark(MGopt.java:198)
	at NPB_MPJ.MGopt.main(MGopt.java:113)

-------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code.. Per user-direction, the job has been aborted.
-------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[36119,1],0]
  Exit code:    1
--------------------------------------------------------------------------
===============================================================================
End: Sat May 31 07:40:16 PDT 2014
